### PR TITLE
add M30 1.10 firmware

### PR DESCRIPTION
**Pull request by cbxbiker61**

### CODE REVIEW

File: releases/M30/1.10/m30.metainfo.xml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      <description>
+        <p>
+          Enhanced the stability of the Bluetooth pairing.
+        </p>
+      </description>
+    </release>
+  </releases>
+  <screenshots>
+    <screenshot type="default">
+      <image type="source">https://raw.githubusercontent.com/hughsie/8bitdo-firmware/master/screenshots/M30.png</image>
```
Comment (hughsie) [inline releases/M30/1.10/m30.metainfo.xml]: I don't see a file there, do you need a hand drawing it?

Comment (cbxbiker61) [reply releases/M30/1.10/m30.metainfo.xml]: Yes, that would be great.  Sorry i forgot to mention the missing png.

Comment (hughsie) [reply releases/M30/1.10/m30.metainfo.xml]: I've added it there.

File: releases/M30/1.10/m30.metainfo.xml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+      <caption>Unplug the controller, hold down L+R+START for 3 seconds until both LEDs are flashing then reconnect the controller.</caption>
+    </screenshot>
+  </screenshots>
+
+  <!-- only newer versions of fwupd know about the new VID -->
+  <requires>
+    <id compare="ge" version="0.9.3">org.freedesktop.fwupd</id>
```
Comment (hughsie) [inline releases/M30/1.10/m30.metainfo.xml]: This needs to be `1.2.7`

**Pull request merged**